File: pfProject_CS/TrialProject/TrialProject/src/PfGrid.cs

using System;
using System.Collections.Generic;
using System.Text;

namespace TrialProject.src {
    public class GridTag{
        public static string typeLand = "Land";
        public static string attributionEmpty = "Empty";
    }
    public class PixelGrid {
        private string type;
        private string attribution;
        private int value;
        public PixelGrid() { }
        public PixelGrid(String ptype = "Land", String attribution = "empty", int value = 1){
            Type = ptype;
            Attribution = attribution;
            Value = value;
        }
        public string Type { get => type; set => type = value; }
        public string Attribution { get => attribution; set => attribution = value; }
        public int Value { get => value; set => this.value = value; }

        private string Dict()
        {
            return "abc";
        }
    }
}

File: pfproject_py/pfgrid.py
# !/usr/bin/env python3
# -*- coding:utf-8 -*-
__author__ = 'Zhiquan Wang'
__date__ = '2018/7/20 22:01'

import taginfo


class GridTag(object):
    type_land = 'land'
    attribution_empty = 'empty'


class PixelGrid(object):

    def __init__(self, *, ptype=GridTag.type_land, attribution='empty', value=1):
        self.__type = ptype
        self.__attribution = attribution
        self.__value = value

    @property
    def type(self):
        return self.__type

    @type.setter
    def type(self, _t):
        self.__type = _t

    @property
    def attribution(self):
        return self.__attribution

    @attribution.setter
    def attribution(self, _a):
        self.__attribution = _a

    @property
    def value(self):
        return self.__value

    @value.setter
    def value(self, _v):
        self.__value = _v

    def __dict__(self):
        return {taginfo.JsonAttribute.pfg_type: self.__type,
                taginfo.JsonAttribute.pfg_attribution: self.__attribution,
                taginfo.JsonAttribute.pfg_value: self.value}
